beast/observationmodel/noisemodel/toothpick.py

```diff
@@ -331,9 +335,9 @@ def interpolate(self, sedgrid, progress=True):
         compl = np.empty((N, M), dtype=float)
 
         if progress is True:
-            it = Pbar(desc='Evaluating model').iterover(range(M))
+            it = Pbar(desc='Evaluating model').iterover(list(range(M)))
```

Comment:
Pbar can take the number of iterations (here M) as parameter so you don't have to instanciate the list. (useful when it gets big)